apps/api/src/database/db.ts

```diff
@@ -133,6 +135,10 @@ export async function dropAll() {
 }
 
 export async function migrateDb() {
```

Comment:
this isn't needed, right?
1. it looks like `baseline.ts` will call `applyBaselineIfNeeded` before it calls this file's `migrateDb`
2. plus, i dont think we want to alter the core logic (even if it's small) for this one-time, manual migration

Reply:
Ah yes - forgot to take that out when trying to ensure consistent behavior between prepping the production environment and using in dev

Reply:
@mzkrasner we still wanna remove this from `db.ts`, right?